src/views/add_challenge.hbs

```diff
@@ -4,8 +4,24 @@
     <h1>Create a new challenge</h1>
   </header>
 
-  <form action="POST">
-    <label for=""></label>
+  <form name="addChallenge" action="/add-challenge" method="POST">
+    <label for="title">Challenge Title</label>
+    <input name="title" id="title" type="text" placeholder="Please type a title">
+    <label for="categoryId">Please select a category</label>
+    <select name="categoryId" id="categoryId">
```

Comment:
let's add an empty field in the dropdown list for page load - maybe with message saying please select an item from the list below?? 🐳